File: Heat/Extensions/AutomapperExtensions.vb
Imports System.Runtime.CompilerServices
Imports AutoMapper

Public Module AutomapperExtensions
    <Extension> _
    Public Sub Bidirectional(Of TSource, TDestination)(this As IMappingExpression(Of TSource, TDestination))
        Mapper.CreateMap(Of TDestination, TSource)()
    End Sub






End Module

File: Heat.ConvertedToC#/Extensions/AutomapperExtensions.cs
using AutoMapper;
namespace Heat
{

    public static class AutomapperExtensions
	{

		public static void Bidirectional<TSource, TDestination>(this IMappingExpression<TSource, TDestination> @this)
		{
			Mapper.CreateMap<TDestination, TSource>();
		}






	}
}
